Specify user on export kubecfg (#9280)
kubernetes/kops: docs/releases/1.19-NOTES.md

# Significant changes

## Changes to kubernetes config export

> Kops will no longer automatically export the kubernetes config on `kops update cluster`. In order to export the config on cluster update, you need to either add the `--user <user>` to reference an existing user, or `--admin` to export the cluster admin user. If neither flag is passed, the kubernetes config will not be modified. This makes it easier to reuse user definitions across clusters should you for example use OIDC for authentication.

Reviewer: ```suggestion
Kops will no longer automatically export the kubernetes config on `kops update cluster`. In order to export the config on cluster update, you need to either add the `--user <user>` to reference an existing user, or `--admin` to export the cluster admin user. If neither flag is passed, the kubernetes config will not be modified. This makes it easier to reuse user definitions across clusters should you, for example, use OIDC for authentication.
```

> Similarly, `kops export kubecfg` will also require that either the `--admin` or `--user` flag is passed if the context does not already exist.

Reviewer: ```suggestion
Similarly, `kops export kubecfg` will also require passing either the `--admin` or `--user` flag if the context does not already exist.
```

> `kops create cluster --yes` implies exporting the admin user together with rest of the cluster config equal to existing behaviour.

Reviewer: ```suggestion
`kops create cluster --yes` exports the admin user along with rest of the cluster config, as is existing behaviour.
```
